core/mapreduce/src/main/java/mil/nga/giat/geowave/mapreduce/operations/ConfigHDFSCommand.java

```diff
@@ -0,0 +1,82 @@
+package mil.nga.giat.geowave.mapreduce.operations;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.ParameterException;
+import com.beust.jcommander.Parameters;
+
+import mil.nga.giat.geowave.core.cli.annotations.GeowaveOperation;
+import mil.nga.giat.geowave.core.cli.api.Command;
+import mil.nga.giat.geowave.core.cli.api.DefaultOperation;
+import mil.nga.giat.geowave.core.cli.api.OperationParams;
+import mil.nga.giat.geowave.core.cli.operations.config.ConfigSection;
+import mil.nga.giat.geowave.core.cli.operations.config.options.ConfigOptions;
+
+@GeowaveOperation(name = "hdfs", parentOperation = ConfigSection.class)
+@Parameters(commandDescription = "Create a local configuration for HDFS")
+public class ConfigHDFSCommand extends
+		DefaultOperation implements
+		Command
+{
+
+	private static final String HDFS_DEFAULTFS_PREFIX = "hdfs.defultFS";
+	private static final String HDFS_DEFAULTFS_URL = HDFS_DEFAULTFS_PREFIX + ".url";
+
+	@Parameter(description = "<HDFS DefaultFS URL> (HDFS hostname:port or namenode HA nameservice, eg: sandbox.hortonworks.com:8020 )")
```

Comment:
parentheses actually aren't going to work well here (unfortunately we probably should delete whats in parentheses for the services to work)